phpcs.xml

```diff
@@ -228,17 +228,6 @@
     -->
     <rule ref="SlevomatCodingStandard.PHP.ShortList"/>
 
-    <!--
-        Requires arrow functions.
-        https://github.com/slevomat/coding-standard/#slevomatcodingstandardfunctionsrequirearrowfunction-
-    -->
-    <rule ref="SlevomatCodingStandard.Functions.RequireArrowFunction">
```

Comment:
Is there any reason why this got dropped?

Reply:
After changing the phpcs PHP version to 70400 this rule started to activate. I thought it produced messier code. The writer should be able to determine whether to use a short closure or traditional closure.

Reply:
For a lot of people it's a matter of habit (arrow functions appeared just a couple of years ago), so I'd keep it as a nudge to use arrow functions.